Localize custom block used alert

apple502j opened the pull request:
### Resolves
Resolves #2050

### Proposed Changes
Adds a l10n message for trying to delete custom blocks in use

### Reason for Changes
Localization!

### Test Coverage
Tests FAIL, and it's expected: https://github.com/LLK/scratch-blocks/wiki/Localization

adroitwhiz committed:
Use internal color slider HSB for slider callbacks

adroitwhiz committed:
Use internal color slider HSB for slider callbacks

chrisgarrity commented on core/block_dragger.js:
```diff
@@ -282,7 +282,7 @@ Blockly.BlockDragger.prototype.endBlockDrag = function(e, currentDragDeltaXY) {
           if (!Blockly.Procedures.getDefineBlock(procCode, ws)) {
             // TODO:(#1151)
-            alert('To delete a block definition, first remove all uses of the block');
+            alert(Blockly.Msg.PROCEDURE_USED);
```
This will only get the pop-up if you drag the definition off the workspace. There is also a version in the context menu procedure: https://github.com/LLK/scratch-blocks/blob/a355ca0b6353cbcfa906472d01a21e9940a390c9/blocks_vertical/vertical_extensions.js#L178-L180

Also, I'm not quite sure whether these are going to get dynamically updated if the language changes.

chrisgarrity reviewed (changes requested):
There's a second place that needs to change. 

Also, before this can be merged it will need to be rebased on the latest develop branch. I'll need to push a commit that pulls all the translations for the new string. Since we recently added Armenian, I can't easily do that without introducing conflicts.

apple502j committed:
Localize custom block used alert

Commit:
[skip ci] Update translations from transifex

Commit:
[skip ci] Update translations from transifex

adroitwhiz committed:
Merge pull request #2066 from adroitwhiz/color-slider-hsb

Use FieldColourSlider's internal hue_, saturation_, brightness_ in slider callbacks

Commit:
[skip ci] Update translations from transifex

Commit:
[skip ci] Update translations from transifex

Commit:
on Windows, use google-closure-compiler.cmd

The `subprocess.Popen` command can't run a program without an extension
on Windows, so this change adds the extension explicitly. Adding
`shell=True` also works on Windows but seems to break building on Linux.

Commit:
on Windows, use google-closure-compiler.cmd

The `subprocess.Popen` command can't run a program without an extension
on Windows, so this change adds the extension explicitly. Adding
`shell=True` also works on Windows but seems to break building on Linux.

chrisgarrity committed:
Update l10n dependency and pull Mongolian

chrisgarrity committed:
Update l10n dependency and pull Mongolian

chrisgarrity committed:
Merge pull request #2079 from chrisgarrity/add-mongolian

Update l10n dependency and pull Mongolian

cwillisf committed:
Merge pull request #2077 from cwillisf/use-closure-cmd

on Windows, use google-closure-compiler.cmd

chrisgarrity committed:
update translations with new string.

chrisgarrity committed:
Merge branch 'localize-custom-block' of https://github.com/apple502j/scratch-blocks into apple502j-localize-custom-block

chrisgarrity reviewed (approved):
Sorry for the additional commits that got pulled in when I tried to rebase/update strings (we'd added another language). The tests should pass now. I'll merge once that's confirmed (they ran locally for me).
Thanks!
